Sources/FioriARKit/ARCards/Models/CardItemModel.swift

```diff
@@ -29,13 +29,32 @@ public protocol IconComponent {
     var icon_: Image? { get }
 }
 
+public struct Vector3: Codable {
+    public var x: Float
+    public var y: Float
+    public var z: Float
+    
+    static let zero = Vector3(x: .zero, y: .zero, z: .zero)
```

Comment:
Clarification question: shouldn't this be `public` access level?

Reply:
This is a tentative solution to get the UI functional primarily to show if in the attachment cell if the annotation has been pinned. I suspect that this will be reworked as we work on the data model for E2E.